Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -326,7 +310,7 @@ void doPublishOdom() {
   double vth = (vr - vl)/(2 * TRACK_RADIUS);
   
   // Calculate the deltas since
-  double dt = current_time.toSec() - last_odom_publish_time.toSec();
+  double dt = current_time.toSec() - last_raw_odom_publish_time.toSec();
```

Comment:
Yeah, one thing I have noticed is that the Arduino IDE is not class friendly. It should be easier to create classes in the project directory and then migrate them to libraries if you want. I end up creating the .h and .cpp files by hand and then I can edit them in the IDE (yes, I have plans for better organized code through refactoring to classes).